patch/0042-add-a7020-comexp-dts.patch

```diff
@@ -0,0 +1,169 @@
+From 5858434dd9a7301bd90730087b527b684a64ec2a Mon Sep 17 00:00:00 2001
+From: gilt <gilt@marvell.com>
+Date: Mon, 26 Aug 2019 14:58:10 +0300
+Subject: [PATCH 5/6] add a7020-comexp dts
+
+Change-Id: If4d6e2b704aad1616ad91871282e678eabb400dd
```

Comment:
If this has been sent upstream, please add the URL or git hash in the Linux repository.